libs/sdk-ui-kit/src/ChartSorting/ChartSortingDropdown.tsx

```diff
@@ -0,0 +1,68 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { IntlShape } from "react-intl";
+import { ISortItem } from "@gooddata/sdk-model";
+
+import { AttributeDropdown } from "./AttributeDropdown/AttributeDropdown";
+import { IAvailableSortsGroup, IBucketItemNames } from "./types";
+
+interface ChartSortingProps {
+    currentSort: ISortItem[];
+    availableSorts: IAvailableSortsGroup[];
+    bucketItemNames: IBucketItemNames;
+    intl: IntlShape;
```

Comment:
You can skip explicit `intl` prop by setting up interface as `interface ChartSortingProps extends WrappedComponentProps {` from `import { WrappedComponentProps } from "react-intl";`

Reply:
Maybe it's meant for `injectIntl` wrapped components however.

Reply:
Yes, `injectIntl` is on parent component - `ChartSorting` and passed down.